pkg/config/validation/validation.go

```diff
@@ -3699,6 +3699,9 @@ func validateTelemetryAccessLogging(logging []*telemetry.AccessLogging) (v Valid
 			continue
 		}
 		if l.Filter != nil {
+			if len(l.Providers) == 0 {
+				v = appendWarningf(v, "filter with empty providers will be ignored")
```

Comment:
Doesn't empty provider mean "use the default"?

seems like the config in the example was correct, just not what the user intended?

Reply:
good catch

```
// Optional. Name of providers to which this configuration should apply.
// If a provider is not specified, the [default logging
// provider][istio.mesh.v1alpha1.MeshConfig.default_providers.] will be used.
```

can we get meshconfig in webhook server?
or just change the messge to `filter with default provider maybe ignored` ?

Reply:
No, we cannot. IMO this is just user error, the config is entirely valid.

Reply:
another option is check meshconfig in webhook?

Reply:
While we cannot do that in general (validation is supposed to be stateless), it also is not fully accurate I think? Since provider can be inherited from a parent Telemetry as well, right?

Reply:
yes, so the best way is improving tests and docs.